Makefile

```diff
@@ -30,16 +30,17 @@ install:
 		mknod -m 666 $(DESTDIR)/dev/urandom c 1 9
 	# copy static files verbatim
 	/bin/cp -a static/* $(DESTDIR)
-	/bin/cp $(CRAFT_PART_INSTALL)/../../probert-deb/install/*.deb $(DESTDIR)/tmp
-	/bin/cp $(CRAFT_PART_INSTALL)/../../consoleconf-deb/install/*.deb $(DESTDIR)/tmp
+	mkdir -p $(DESTDIR)/install-data
+	/bin/cp -r $(CRAFT_STAGE)/local-debs $(DESTDIR)/install-data/local-debs
 	# customize
 	set -ex; for f in ./hooks/[0-9]*.chroot; do \
-		/bin/cp -a $$f $(DESTDIR)/tmp && \
-		if ! chroot $(DESTDIR) /tmp/$$(basename $$f); then \
+		/bin/cp -a $$f $(DESTDIR)/install-data/hook && \
```

Comment:
Maybe a nitpick, but if you call all of the hooks "hook", `$0` won't reflect the original name, which is something sometimes useful (printing traces, etc.)